spec/jasmine-fixture-spec.coffee

```diff
@@ -1,7 +1,9 @@
 describe "jasmine.fixture", ->
+  Then "no emmet global", -> `(1, eval)('this')`.hasOwnProperty('emmet') == false
 
   EXAMPLES = [
      'article'                                                             #<article></article>
+     'backlog-item'                                                        #<backlog-item></backlog-item>
```

Comment:
was this an edge case previously not supported by the old parser? (custom elements?)